app/uk/gov/hmrc/trusts/controllers/GetTrustController.scala

```diff
@@ -55,7 +57,23 @@ class GetTrustController @Inject()(identify: IdentifierAction,
   def get(utr: String): Action[AnyContent] = (ValidateUTRAction(utr) andThen identify).async {
     implicit request =>
 
-      desService.getTrustInfo(utr, request.identifier).map {
+      desService.getTrustInfo(utr, request.identifier).flatMap {
```

Comment:
Default behaviour should return user answers with no deltas. New endpoint needed.